src/__forks__/Relay.js

```diff
@@ -16,10 +16,6 @@
 var RelayDefaultNetworkLayer = require('RelayDefaultNetworkLayer');
 var RelayPublic = require('RelayPublic');
 
-// By default, assume that GraphQL is served at `/graphql` on the same domain.
-// $FlowFixMe(>=0.16.0)
-RelayPublic.injectNetworkLayer(new RelayDefaultNetworkLayer('/graphql'));
```

Comment:
I have removed the injection here because the default network layer is now injected in the constructor of `RelayContext`.